spec/catalog.yaml

```diff
@@ -261,6 +295,18 @@ components:
           default: { }
           nullable: true
 
+    GetTableResult:
+      description: |
+        Result used when a table is successfully loaded.
+      type: object
+      required:
+        - metadata
+      properties:
```

Comment:
what do you expect to return in the properties? I thought the response would just have the location of the table

Reply:
My thought is that we return a map for further extension.  Currently, maybe, location, and some metadata, e.g. create_at and so on.. future, maybe latest_version...

Reply:
I see. Then I am more curious about how we should do CreateTable. 

Typically there is a consistency with what is used to CreateTable, vs what is returned in GetTable, this has been true in HMS, IRC and also vendor commercial APIs like Glue, Unity, etc. It might not be exactly the same, but usually mostly the same, so basically you get what you create.

Also, it is true that Table typically has a free-form properties field, but important information are usually top level fields, especially the ones you mentioned like location, created_at, latest_version, etc. For example take a look at Unity TableInfo definition: https://github.com/unitycatalog/unitycatalog/blob/main/api/all.yaml#L1559

Reply:
Let‘s add some clear fields and leave a general map field.